blueocean-dashboard/pom.xml

```diff
@@ -17,7 +17,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>sse-gateway</artifactId>
-            <version>1.1</version>
+            <version>1.2-SNAPSHOT</version>
         </dependency>
     </dependencies>
 
```

Comment:
@cliffmeyers Right, I'll do a proper release of that plugin (and he npm package) once we agree this is ok (ala what I said in the description).